src/applications/mhv/secure-messaging/reducers/folders.js

```diff
@@ -1,6 +1,10 @@
 import { Actions } from '../util/actionTypes';
 
 const initialState = {
+  /**
+   * The current in-focus folderId, i.e. the folderId of the folder being viewed by the user
+   */
+  folderId: undefined,

```

Comment:
not convinced we need to create this as we already have this captured in a `folder` object in this same reducer. Or is this somehow different?

Reply:
yes you are correct, I went ahead and removed it